scripts/resolve_api_audit.py

```
@@ -0,0 +1,248 @@
+    item = ctx["first_clip"]
+    if not item:
+        return "skipped", "no clip on V1 to probe"
+    if not hasattr(item, "AddMarker") or not hasattr(item, "DeleteMarkerAtFrame"):
+        return "limited", "AddMarker present, DeleteMarkerAtFrame missing"
+    frame = 1
+    added = item.AddMarker(frame, "Red", "buttercut_audit_probe", "buttercut audit", 1)
+    if added:
+        try:
+            item.DeleteMarkerAtFrame(frame)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)